web/api/events.js

```diff
@@ -137,6 +168,51 @@ exports.register = function (server, eOptions, next) {
       auth: auth.apiUser,
       description: 'Update applications by bulk',
       tags: ['api', 'events'],
+      validate: {
+        // Unknow used again due to dirty payload by angular scope
+        payload: {
```

Comment:
ticketName, ticketType, should, even tho they shouldn't because it shouldn't be handled by the front.
`created` is not sent from the front-end, neither is `status`, at least at creation

Reply:
dob can't be required, it's not set when dojo-admin creates a new application